lib/tasks/hmrc_manuals_go_live.rake

```diff
@@ -0,0 +1,9 @@
+desc "Replace beta phase on HMRC manuals and manual sections with live"
+task hmrc_manuals_go_live: :environment do
+  schema_name = %w[hmrc_manual hmrc_manual_section]
+  items = Edition.live.where(schema_name:, phase: "beta").where.not(content_store: nil)
```

Comment:
I think `where.not(content_store: nil)` is redundant, because `Edition.live` becomes the SQL statement `WHERE "editions"."content_store" = 'live'`. But it won't do any harm.